src/System.Data.SqlClient/src/System.Data.SqlClient.csproj

```diff
@@ -11,16 +11,19 @@
     <PlatformTarget>AnyCPU</PlatformTarget>
     <PackageTargetFramework>dotnet5.5</PackageTargetFramework>
     <UsePackageTargetRuntimeDefaults>true</UsePackageTargetRuntimeDefaults>
+    <ProjectGuid>{D4550556-4745-457F-BA8F-3EBF3836D6B4}</ProjectGuid>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(TargetsWindows)' != 'true' ">
     <DefineConstants>$(DefineConstants);MANAGED_SNI</DefineConstants>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(TargetsWindows)' == 'true' ">
```

Comment:
Refer to https://github.com/dotnet/corefx/blob/master/src/System.Net.Security/src/System.Net.Security.csproj#L134
This depicts how to compile different files for platforms.